core/frontend/src/components/vehiclesetup/PwmSetup.vue

```diff
@@ -269,6 +300,11 @@ export default Vue.extend({
         .map((_, i) => data[`servo${i + 1}_raw`])
     },
   },
+  watch: {
+    is_armed() {
+      this.desired_armed_state = this.is_armed
```

Comment:
please add a comment to explain the reason of the watch, since it's not clear that the idea is to track desired behavior change from others software like cockpit.